README.md

```diff
@@ -1,6 +1,44 @@
-# library_project_scripting
-A temporary repository that stores library data from ArchivesSpace as JSON files.
+# RAC Library Discovery Catalog System
 
-Includes Resource records, associated Subjects, Agents, and  Top Container records.
+A Jekyll web application which provides the front-end user interface for the online discovery of bibliographic materials at the Rockefeller Archive Center.
 
-The sample_data directory includes a small sample set of data that we used for testing our discovery system.
+## Dependencies
+
+* [Python 3](https://www.python.org/download/releases/3.0/)
```

Comment:
Do we need a specific version of Python 3? 3.6+?

Reply:
As far as I'm aware, the only portion requiring Python is the `makePages.py` script. So, unless I'm forgetting something, we don't need a specific version.

Reply:
Well in that case I think it would be wise to at least specify which version we've tested this on. You've linked to 3.0 here, which I don't think we've tested on.